Tutorial.md

```diff
@@ -0,0 +1,178 @@
+# Welcome to the BiobakeryUtils tutorial
+
+## Using BiobakeryUtils with [MetaPhlAn](https://github.com/biobakery/MetaPhlAn/wiki/MetaPhlAn-3.0) files
+
+### Input file format
+
+BiobakeryUtils accepts `.tsv` files outputted from MetaPhlAn.
+Files can either contain a merged table or one or multiple single table(s).
+Example files that are used in this tutorial can be found in [BiobakeryUtils.jl/test/files/](https://github.com/BioJulia/BiobakeryUtils.jl/tree/main/test/files).
+
+### Creating CommunityProfiles
+
+A CommunityProfile can be created from MetaPhlAn file(s) using `metaphlan_profile` and `metaphlan_profiles`.
+
+From one file with a single table:
```

Comment:
Make a note somewhere that currently, this only works for profiles generated with metaphlan v3